docs/mem_quick_start.ipynb

```diff
@@ -0,0 +1,597 @@
+   "source": [
+    "embedding_list = await embedding_service.aembed_query(\"test query\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The `AlloyDBEmbeddings` class can be used as the embedding service while creating the `AlloyDBVectorStore`.\n",
+    "\n",
+    "You can find a detailed sample notebook on `AlloyDBVectorStore` [here](vector_store.ipynb)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from langchain_google_alloydb_pg import AlloyDBVectorStore\n",
+    "import uuid\n",
+    "from langchain_core.documents import Document\n",
+    "\n",
+    "texts = [\"foo\", \"bar\", \"baz\", \"boo\"]\n",
+    "VECTOR_SIZE = 768  # For textembeddding-gecko@003 model\n",
+    "ids = [str(uuid.uuid4()) for i in range(len(texts))]\n",
+    "metadatas = [{\"page\": str(i), \"source\": \"google.com\"} for i in range(len(texts))]\n",
+    "docs = [\n",
+    "    Document(page_content=texts[i], metadata=metadatas[i]) for i in range(len(texts))\n",
+    "]\n",
+    "\n",
+    "await engine.ainit_vectorstore_table(\n",
+    "    table_name=\"vector_store_table\",\n",
+    "    vector_size=VECTOR_SIZE,\n",
+    "    overwrite_existing=True,\n",
+    ")\n",
+    "\n",
+    "vs = await AlloyDBVectorStore.create(\n",
+    "    engine,\n",
+    "    embedding_service=embedding_service,\n",
+    "    table_name=\"vector_store_table\",\n",
+    ")\n",
+    "\n",
+    "await vs.aadd_documents(docs, ids=ids)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### To use `AlloyDBVectorStore` functionalities"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To use asimilarity_serach_by_vector / amax_marginal_relevance_search_vector / amax_marginal_relevance_search_vector,\n",
+    "\n",
+    "You can generate the embeddings for a query using the `AlloyDBEmbeddings` class and use that with the vector store."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "search_embedding = embedding_service.embed_query(\"foo\")\n",
+    "results = await vs.asimilarity_search_by_vector(search_embedding)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can also use the other functions such as:\n",
+    "* asimliraity_search\n",
+    "* asimliraity_search_score\n",
+    "* amax_marginal_relevance_search\n",
+    "\n",
+    "And more which are a part of the `AlloyDBVectorStore` class\n",
+    "to use the provided `AlloyDBEmbedding` instance implicitly."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "results = await vs.asimilarity_search(\"foo\", k=1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Removing a Model\n",
```

Comment:
Actually, this needs to be towards the end.. so that a user following the notebook has a model registered.. plays around with the model and drops it towards the end.

We can add a disclaimer that model can be dropped, in the section where its created.